### PR TITLE
Erica Thompson: WIP

### PR DESCRIPTION
**Pull request by EricaThompson**
- [x] fizzbuzz
- [x] strings
- [ ] pointers
- [ ] malloc
- [ ] structs
- [ ] queue
- [ ] quicksort
- [ ] [Stretch Goal] hangman

### CODE REVIEW

File: fizzbuzz/fizzbuzz.c [hunk 1]
```diff
@@ -10,7 +10,27 @@
 */
 int fizzbuzz(int n)
 {
-
+    int counter = 0;
+    int i = 0;
+    for(i = n; i > 0; i--)
+    {
+        if(i % 3 == 0 && )
```
Comment (wvandolah) [inline fizzbuzz/fizzbuzz.c]: missing something after `&&`

File: fizzbuzz/fizzbuzz.c [hunk 2]
```diff
@@ -10,7 +10,27 @@
...
+        {
+            printf("Fizz\n");
+            return 0;
+        }
+        else if(i % 5 == 0)
+        {
+            printf("Buzz\n")
```
Comment (wvandolah) [inline fizzbuzz/fizzbuzz.c]: need `;` at the end of this line

File: strings/strings.c [hunk 3]
```diff
@@ -20,7 +26,14 @@ int string_length(char s[])
...
 {
-
+    int count = 0;
+    for(int i = string_length(s) - 1; i >= 0; i--)
+    {
+        rv[count] = s[i];
+        count++;
+    } 
+    rv[string_length(s)] = '\0';
+    return rv;
```
Comment (wvandolah) [inline strings/strings.c]: 👍

File: fizzbuzz/fizzbuzz.c (hunk 1 above)
Comment (EricaThompson) [reply fizzbuzz/fizzbuzz.c]: This isn't complete.

File: fizzbuzz/fizzbuzz.c (hunk 2 above)
Comment (EricaThompson) [reply fizzbuzz/fizzbuzz.c]: This is WIP.

File: strings/strings.c (hunk 3 above)
Comment (EricaThompson) [reply strings/strings.c]: 👍

**Pull request closed without merging**